doc/Roadmap.rst

```diff
@@ -1,158 +1,156 @@
+    like hover information, legends, and colorbars. There is already extensive
+    support for rendering Datashader output in Bokeh, along with some support in
+    Matplotlib and Plotly. The core team is gradually further improving Bokeh's
+    support, with nearly all of Datashader's shading functionality now or soon
+    available interactively in Bokeh, and similar support can be added to other
+    libraries by their users or contributors.
+
+3.  **Support for managing annotations**: Users of a plot or dashboard often
+    notice important features of the data on display, but don't have a convenient
+    way to capture or communicate that insight. The core team is developing
+    annotations as a first-class component the HoloViz tools, easily added onto
+    plots and dashboards so that users can denote regions of interest in some
+    space (e.g. a time range on a timeseries plot), storing, restoring, and
+    displaying that information in editable form.
+
+Other targets needed in HoloViz or in packages designed to work well with
+HoloViz include:
+
+1.  **More extensive documentation about deployment**: HoloViz includes a
+    tutorial on `deployment using Bokeh server
+    <http://holoviz.org/tutorial/13_Deploying_Bokeh_Apps.html>`__, but there are
+    many other ways to set up live Python-backed plots and Panel dashboards or
+    apps, including creating a Flask server or embedding into Django, along with
+    deploying via Heroku or Google Cloud and AWS or via the Anaconda Enterprise
+    platform or on `MyBinder <https://mybinder.org>`__. Documenting and testing
+    these possibilities takes time and effort, and any contributions or examples
+    that will help people get started and decide between the alternatives would
+    be very helpful.
+
+2.  **Bokeh WebGL support**: Bokeh provides good support for working
+    interactively with small datasets using an HTML Canvas (client-side
+    interactivity), and when combined with Datashader it can handle enormous
+    datasets by pre-rendering them to images in Python (server-side). However, in
+    between these two extremes there is a range of data sizes that could be
+    addressed well by the client-side WebGL technology. Bokeh has long included
+    some WebGL support, but improvements in 2021-2022 have made the WebGL support
+    much more solid and usable, and updating HoloViews to use that could bring
+    big performance improvements for midsize plots.
+
+3.  **New and improved workflow examples in biomedical and Earth science,
+    especially utilizing machine learning**: Demonstrations for handling and
+    visualization of various large, multidimensional datasets from data-intensive
+    research domains, including on HPC systems.
+
+4.  **Better deployment and exploration of large-scale visualizations**: For
+    instance, with improvements to automatic generation of tiles for exploration
+    of large datasets using standard web servers.
```

Comment:
Maybe link to the last version of those examples on Datashader that got deleted, to point people to what to pick up?

Reply:
this? https://datashader.org/tiling.html

Reply:
The better link would be https://github.com/holoviz/datashader/blob/main/examples/tiling.ipynb , but for some reason I thought it had been deleted recently.

Reply:
link to nb added